test/bdd/fixtures/adapter-rest/manifest-config/outputdescriptors.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
why this file is being added in bddtests? I dont see you are using it anywhere in bdd tests.

Reply:
I will delete it for now , will add when i will have bdd test implementation after the follow up pr. #561

Reply:
I am using it docker-compose.yml file ADAPTER_REST_OUTPUT_DESCRIPTORS_FILE=/etc/manifest-config/outputdescriptors.json

Reply:
Yes, but did you use in any of the bddtests? if not then you dont need this for in bddtests fixtures.